motorSupportApp/src/devMotorAxis.cpp

```diff
@@ -399,24 +399,21 @@ void devMotorAxis::getInteger(std::string pvSuffix, epicsInt32* pvalue, const st
   * \return The status code from doing the poll.
   */
 asynStatus devMotorAxis::poll(bool *moving) {
-    asynStatus comStatus = asynSuccess;
-    int nowMoving = 0;
 	st_axis_status_type st_axis_status;
 	memset(&st_axis_status, 0, sizeof(st_axis_status));
-	
-	// The comms error neeeds to be set to 0 to start to force the new error on init
-	setIntegerParam(pC_->motorStatusCommsError_, 0);
 
 	try {		
 		// Go and get all the values from the device
 		pollAll(&st_axis_status);
-
+    // The comms error neeeds to be set to 0 to start to force the new error on init
+	  setIntegerParam(pC_->motorStatusCommsError_, 0);
```

Comment:
Not sure I understand this comment - would it be possible to expand/clarify it slightly?

Reply:
I think it is now redundant - will remove